### ISSUE TITLE
Update to 1.21.1

### ISSUE THREAD

**Issue by MimzyFang** [Kikugie/elytra-trims#81]
### Description

while the mod is probably compatible with 1.21.1 since those versions are super close to each other it's marked as only compatible with 1.21 so it would be nice if you would build it marking it to be compatible with 1.21 and 1.21.1 if posible

```
 - Replace mod 'Elytra Trims' (elytratrims) 3.4.4 with any version that is compatible with:
		 - minecraft 1.21.1
```

**Comment by kikugie** [comment]
Will be available soon in v3.5, after a couple more bugfixes

**Comment by kikugie** [comment]
Resolved in 3.5.0
